lib/ansible/modules/packaging/os/apt.py

```diff
@@ -545,9 +547,22 @@ def install(m, pkgspec, cache, upgrade=False, default_release=None,
         if rc:
             status = False
             data = dict(msg="'%s' failed: %s" % (cmd, err), stdout=out, stderr=err, rc=rc)
-        return (status, data)
     else:
-        return (True, dict(changed=False))
+        status = True
+        data = dict(changed=False)
+
+    if not build_dep:
+        mark_installed_manually(m, package_names)
+
+    return (status, data)
+
+
+def mark_installed_manually(m, packages):
+    apt_mark_cmd_path = m.get_bin_path("apt-mark", required=True)
+    cmd = "%s manual %s" % (apt_mark_cmd_path, ' '.join(packages))
```

Comment:
"manual" is a recen' occurrence, for older versions it was 'unmarkauto' iirc

Reply:
Thanks, added a fallback for that case.